### PR TITLE
add sizeref to bubble docs

### PR DESCRIPTION
**Pull request by Kully**
deals with: https://github.com/plotly/documentation/issues/776

### CODE REVIEW

**Comment by Kully** [comment]
Only issue I'm having right now is an error in the R doc saying that I am using raw data:

screen shot 2018-09-11 at 3 55 20 pm

**Comment by priyatharsan** [comment]
@Kully I think the raw data issue is relevant to this: https://github.com/ropensci/plotly/issues/1330

**Comment by Kully** [comment]
I don't think I am using mapbox though, so it may be a different issue

**Comment by priyatharsan** [comment]
Oops! What I meant was the `api_create` is looking for Grid type data and not Raw data. And its happening the same as for the case with mapbox as well.

**Comment by Kully** [comment]
Everything else being equal, setting `sizemode = 'diameter'`doesn't produce the Raw Data Error, but sizemode = 'area' _does_. Even not setting the sizemode param causes the error.

**Comment by Kully** [comment]
only issue is the r doc - for some reason `sizeref` is not behaving the way I want it to. It's producing too small points.

File: _posts/python/maps/bubble-maps/bubble-maps.ipynb [hunk 1]
```diff
@@ -1,250 +1,339 @@
...
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import plotly.plotly as py\n",
+    "import pandas as pd\n",
```
Comment (bcdunbar) [inline _posts/python/maps/bubble-maps/bubble-maps.ipynb]: perhaps add a gap between the two to remain consistent.

```
import plotly.plotly as py

import pandas as pd
```

File: _posts/python/maps/bubble-maps/bubble-maps.ipynb [hunk 2]
```diff
@@ -1,250 +1,339 @@
...
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import plotly.plotly as py\n",
+    "import plotly.graph_objs as go\n",
+    "\n",
+    "import pandas as pd\n",
+    "df = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/2014_ebola.csv')\n",
```
Comment (bcdunbar) [inline _posts/python/maps/bubble-maps/bubble-maps.ipynb]: add a space between import and df =pd.read... line

**Pull request merged**